pkg/server/routes.go

```diff
@@ -54,8 +55,12 @@ func (srv *RegistrationServer) SetupRoutes() error {
 		// /status is something you should always have in any of your services,
 		// please leave it as is.
 		healthService := health.NewHealthCheckService(srv.logger, srv.Config())
-		signupService := signup.NewSignupService(srv.logger, srv.Config())
-
+		signupService, err := signup.NewSignupService(srv.logger, srv.Config())
+		if err != nil {
+			// error creating signup service, bail out.
+			log.Fatal(err)
```

Comment:
Shouldn't we return the error here?

Reply:
This part was restructured by Tina anyway..